instance/templates/instance/haproxy/openedx.conf

```diff
@@ -1,10 +1,10 @@
 {% autoescape off %}
     # Backend configuration for {{ domain }}
-    stick-table type string len 128 size 20k expire 12h
-    stick on req.cook(sessionid)
+    cookie openedx-backend insert postonly indirect
     acl has-authorization req.hdr(Authorization) -m found
     http-request set-header Authorization 'Basic {{ http_auth_info_base64 }}' unless has-authorization
+    option httpchk GET /heartbeat HTTP/1.1
```

Comment:
I apparently got this wrong.  For HTTP version to 1.1, the Host header is mandatory, but we don't provide it, so all health checks return a 400 and all backends are disabled.  The easiest solution is to use the default of HTTP version 1.0 instead, so this line can simply become

    option httpchk /heartbeat

I already tested this by manually editing the configuration of your sandbox on the LB, and it worked fine.